addon/components/session-overview.js

```diff
@@ -207,6 +244,30 @@ export default Component.extend(Publishable, Validations, ValidationErrorDisplay
         });
       }
     },
+    async setPostrequisite(id) {
+      if (! id) {
+        this.set('postRequisite', null);
+        return;
+      }
+      const session = this.get('session');
+      const course = await session.get('course');
+      const sessions = await course.get('sessions');
+      const postRequisite = sessions.findBy('id', id);
+      this.set('postRequisite', postRequisite);
+    },
+
+    changePostrequisite() {
+      const session = this.get('session');
+      const postRequisite = this.get('postRequisite');
+      session.set('postrequisite', postRequisite);
+      session.save();
```

Comment:
Should probably be `await session.save()`.

Reply:
handled.